### ISSUE TITLE
Task list populated with list of Activities indicated for each facility

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#267]
#### Description:

After #237 we need to populate the task list with a list of activity pages based on what the user selects for their facility information each. Each record in the activity table is weighted. The list of activities should be ordered lowest to highest weight.

As an industry reporter, I want to see the task list populated with the list of Activities I indicated for each facility so that I can report my emissions

#### Acceptance Criteria:

Given I am an Industry Reporter 
When I start an Annual Report
Then I want the task list on the left-hand side to display the Activities that I selected for each facility

Given I am an Industry Reporter 
Whenever I view the task list
Then the order of activities is always the same

Given I am an Industry Reporter 
When I click on an activity in the task list
Then I am taken to that activity's form page

#### Development Checklist:

- [x] The list of activities selected for a facility is shown in ascending weighted order in the task list
- [x] Clicking on an activity in the task list takes you to that activity's form page
- [x] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [x] User story is included
- [x] User role and type are identified
- [x] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [x] Design / Solution is accepted by Product Owner
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By

#237 
#311 

#### Notes:

-

**Comment by rdromey** [comment]
@dleard @pbastia to discuss details further based on previous work before refining

**Comment by rdromey** [comment]
@dleard to break out pre-work for this work into a separate ticket.
